src/executor_test.cc

```diff
@@ -293,7 +293,7 @@ logicOp)";
 
   auto color_frame_format = ToStub(engine.get())->GetColorFrameFormat();
   auto depth_frame_format = ToStub(engine.get())->GetDepthFrameFormat();
-  EXPECT_EQ(FormatType::kUnknown, color_frame_format);
+  EXPECT_EQ(FormatType::kB8G8R8A8_UNORM, color_frame_format);
```

Comment:
I'm surprised this is not kR8G8B8A8_UNORM.  (i,e. rgba order instead of bgra)

Reply:
That's the default vkrunner uses: https://github.com/Igalia/vkrunner/blob/master/vkrunner/vr-script.c#L2466